src/waitForImagesToRender.js

```diff
@@ -14,6 +14,16 @@ export default function waitForImagesToRender() {
       .filter(Boolean)
       .map(url => waitForImageToLoad(url));
 
+    document.body.querySelectorAll('*').forEach((element) => {
+      const computedStyle = window.getComputedStyle(element);
+      const bgImage = computedStyle.getPropertyValue('background-image');
+      if (bgImage && bgImage.startsWith('url(')) {
```

Comment:
This doesn't look like it will work if using multiple background images. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Background_and_Borders/Using_CSS_multiple_backgrounds